src/Analysis/Ast/Test/Microsoft.Python.Analysis.Tests.csproj

```diff
@@ -42,6 +42,9 @@
     <ProjectReference Include="..\..\Core\Impl\Microsoft.Python.Analysis.Core.csproj" />
     <ProjectReference Include="..\Impl\Microsoft.Python.Analysis.csproj" />
   </ItemGroup>
+  <ItemGroup>
+    <Folder Include="Properties\" />
```

Comment:
Do we need this?

Reply:
There is `AssemblyInfo` there with `InternalsVisibleTo("Microsoft.Python.Analysis.Caching.Tests)` for fluent things.

Reply:
Ok